src/codeflare_sdk/cluster/cluster.py

```diff
@@ -248,7 +303,7 @@ def list_all_clusters(
     return clusters
 
 
-def list_all_queued(namespace: str, print_to_console: bool = True):  # pragma: no cover
+def list_all_queued(namespace: str, print_to_console: bool = True):
```

Comment:
Should this function ignore running clusters? And only show Pending clusters?

Reply:
I think it is fine for now, just so that it shows all active AppWrappers/MCAD-managed resources. I think a good change for the future would have the use be able to pass in the "filter" that this function uses internally in a clearer manner, and then actually fix it functionally within by ignoring what doesn't match the filter.

Reply:
That way a user could see all pending requests, all failed requests, all running requests, or any combination.